lib/figaro.rb

```diff
@@ -0,0 +1,63 @@
+require 'yaml'
+require 'erb'
```

Comment:
What of ERB are we using?

Reply:
Oh, I just discovered the conversation at https://github.com/18F/identity-idp/pull/4408#discussion_r523288203 . I assume we could remove this now?

```suggestion
```

Reply:
Yep, thanks